apps/E2E/src/Avatar/specs/Avatar.spec.win.ts

```diff
@@ -34,30 +29,47 @@ describe('Avatar Accessibility Testing', () => {
     await AvatarPageObject.scrollToTestElement();
   });
 
-  it('Validate accessibilityLabel', async () => {
-    await expect(await AvatarPageObject.getPrimaryComponentAttribute(ACCESSIBILITY_LABEL_ATTR)).toEqual(AVATAR_ACCESSIBILITY_LABEL);
+  it('Set accessibilityLabel -> Validate set accessibilityLabel is correct', async () => {
```

Comment:
Nit: Remove "set" in 2nd part of error message 

"Set accessibilityLabel -> Validate accessibilityLabel is correct"